crates/core/src/ring_switch/tests.rs

```diff
@@ -310,26 +312,40 @@ fn commit_prove_verify_piop<U, F, MTScheme, MTProver>(
 	let mut proof = ProverTranscript::<HasherChallenger<Groestl256>>::new();
 	proof.message().write(&commitment);
 
-	let backend = make_portable_backend();
 	let ReducedWitness {
 		transparents: transparent_multilins,
 		sumcheck_claims,
 	} = prove(&system, &committed_multilins, &mut proof, MemoizedData::new()).unwrap();
 
-	let domain_factory = DefaultEvaluationDomainFactory::<B8>::default();
+	let hal = CpuLayer::<Tower>::default();
+	let host_mem_size_committed = committed_multilins.len();
```

Comment:
This is precise, but not necessary. It's just a test -- you can just hardcode a value that's large enough to work and keep rerunning it until it does. Allocating 64 KiB - 1 MiB should be totally fine.